### ISSUE TITLE
Space at bottom of last widget even though Global and Column Widgets space is set to 0

### ISSUE THREAD

**Issue by kirstenray** [pojome/elementor#4452]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Description
After setting the global settings Widget Space to 0 and the column Widget Space to 0.  And also set the padding and margin of the column and section to 0. There is still white space at the bottom of the last widget.

I have tried with the 2017 and 2016 default themes, Generate Press, Astra, and Elementor's own Hello Theme.  All of these themes still show the space at the bottom of the last widget.

I deactivated all plugins but Elementor and Elementor Pro.

## Steps to reproduce

Create a section, with one column.  
Add a header widget.
Add a text editor widget. 
Set the column Widget Space to 0.
Set the global Widget Space to 0.
Set the padding and margins on the section, column, header widget, text editor widget to 0.

## Isolating the problem
- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a default WordPress theme active.
- [X] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```

== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: 5.6.39
	PHP Version: 7.1.14
	PHP Max Input Vars: 1000
	PHP Max Post Size: 8M
	GD Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 4.9.5
	Site URL: https://nicolecyrus.com
	Home URL: https://nicolecyrus.com
	WP Multisite: No
	Max Upload Size: 2 MB
	Memory limit: 40M
	Permalink Structure: /%category%/%postname%/
	Language: en
	Timezone: 0
	Admin Email: kirstensray@gmail.com
	Debug Mode: Inactive

== Theme ==
	Name: GeneratePress
	Version: 2.1.1
	Author: Tom Usborne
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.40 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 2.0.11
		Author: Elementor.com

	Elementor Pro
		Version: 2.0.5
		Author: Elementor.com


== Debug ==
	Errors: There are no errors to display

```
</details>

**Comment by bainternet** [comment]
@kirstenray 

> Set the global Widget Space to 0.

which global widget? the steps don't make sense...

also If the last widget is a Text Widget are you sure the padding is not from the theme's `P` tag style?

**Comment by kirstenray** [comment]
There is a global setting called **Widget Space**, not a global widget.  To see this setting, edit a page, then click on the hamburger menu in the upper left corner of the settings panel, then click on Global Settings.  It is the 3rd setting down.

To see the Widget Space setting for a column, edit a page, click on the layout icon of the column and in the settings panel on the left-hand side of the screen.  In the layout settings, Widget Space is the 3rd settings. 

To see the margin and padding settings, edit a page and click on the layout icon of the column or section and then click the Advanced in the settings panel to the left.

Thank you for asking about the theme adding padding to P tag.  I ran that down.  There is no P tag in the text that I entered when editing the page.  I checked by clicking on the Text tab of the editor.  When I roll over the text widget the blue border that surrounds the widget does not show the text portion creating the space.  The space is beneath the widget boundary.  HOWEVER, when the page is rendered the one line of text renders with a P tag around it.  In looking at the CSS for this element the css below is what is causing the extra space.

```
.site-main .gallery, fieldset legend, p {
    margin-bottom: 1.5em;
}
```
I went to the Wordpress Theme editor and looked in style.css and found this

```
p {
	margin-bottom: 1.5em;
}
```

This does appear to be caused by the theme and not the widget spacing of elementor.
